### PR TITLE
ability to throw params with simple form

### PR DESCRIPTION
**Pull request by skv-headless**
for example

```= f.input :description, as: :wysihtml5, wysihtml5: {locale: 'ru-RU'}```

### CODE REVIEW

**Comment by Nerian** [comment]
Cool thanks.

File: lib/bootstrap-wysihtml5-rails/simple_form/wysihtml5_input.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@ class Wysihtml5Input < SimpleForm::Inputs::TextInput
   def input(wrapper_options)
     idf = "#{lookup_model_names.join("_")}_#{reflection_or_attribute_name}"
     script = template.content_tag(:script, type: 'text/javascript') do
-      "$('textarea[id=#{idf}]').wysihtml5();".html_safe
+      "$('textarea[id=#{idf}]').wysihtml5(#{options[:wysihtml5].to_json});".html_safe
```
Comment (Nerian) [inline lib/bootstrap-wysihtml5-rails/simple_form/wysihtml5_input.rb]: Is it _options_ or _wrapper_options_?

Comment (skv-headless) [inline lib/bootstrap-wysihtml5-rails/simple_form/wysihtml5_input.rb]: I'm not sure that I understood question. For case 
```= f.input :description, as: :wysihtml5, wysihtml5: {locale: 'ru-RU'}```
wysihtml5 in **options**

Comment (Nerian) [inline lib/bootstrap-wysihtml5-rails/simple_form/wysihtml5_input.rb]: Just checking. Excellent :)

**Pull request merged**